### DISCUSSION TITLE
MAA stop working at shard factory

### DISCUSSION THREAD

**Discussion by LipChuanSui** [MaaAssistantArknights/MaaAssistantArknights#7875] [Q&A]
Hi, MAA stop choosign operator when navigate to origium shard. Please help.
![image](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/56187131/c1f865b5-0c64-4149-9cb6-d899c67d714d)

**Comment by ABA2396** [comment]
https://github.com/MaaAssistantArknights/MaaAssistantArknights/issues/new/choose
